src/pybind/mgr/dashboard/controllers/rgw.py

```diff
@@ -1298,3 +1298,34 @@ def get_user_list(self, zoneName=None):
         multisite_instance = RgwMultisite()
         result = multisite_instance.get_user_list(zoneName)
         return result
+
+
+@APIRouter('/rgw/topic', Scope.RGW)
+class RgwTopic(RESTController):
+
+    def create(self, daemon_name=None, owner=None,
+               topic_name: str = '', push_endpoint: Optional[str] = None,
+               OpaqueData: Optional[str] = None, persistent: Optional[bool] = None,
```

Comment:
let's follow the pythonic naming convention.
```suggestion
               opaque_data: Optional[str] = None, persistent: Optional[bool] = None,
```

Reply:
changes are done